### PR TITLE
In 133967227 add race to person new

### PR DESCRIPTION
**Pull request by noraOSI**

### CODE REVIEW

File: app/assets/javascripts/components/people/PersonShowPage.jsx [hunk 1]
```diff
@@ -103,6 +103,13 @@ export class PersonShowPage extends React.Component {
                 ))
             }
           </div>
+          <div className='row gap-top'>
+            <ShowField gridClassName='col-md-4' label='Race'>
+              {
```
Comment (pemaOSI) [inline app/assets/javascripts/components/people/PersonShowPage.jsx]: can it be one line?

File: app/assets/javascripts/Races.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+/*eslint quote-props: ["error", "consistent-as-needed"]*/
+const RACES = Object.freeze({
+  'White': {
+    exclusive: false,
```
Comment (pemaOSI) [inline app/assets/javascripts/Races.js]: "exclusive" needs to be renamed. I had to read the rest of the code to understand it. 
Seems like it should say "white" race have more options to specify.

Comment (noraOSI) [inline app/assets/javascripts/Races.js]: We used "exclusive" to indicate that only one option can be selected, meaning other options should get disabled. Is it more clear if we change it to "exclusive option", "exclusive selection", "isSingleChoice"?

File: app/assets/javascripts/components/people/PersonShowPage.jsx (hunk 1 above)
Comment (noraOSI) [inline app/assets/javascripts/components/people/PersonShowPage.jsx]: done!

**Pull request merged**